frontend/app/src/components/home/CurrentChatMenu.svelte

```diff
@@ -364,13 +363,16 @@
                             </MenuItem>
                         {/if}
                     {/if}
-                    <MenuItem disabled={unreadMessages === 0} on:click={markAllRead}>
-                        <CheckboxMultipleMarked
-                            size={$iconSize}
-                            color={"var(--icon-inverted-txt)"}
-                            slot="icon" />
-                        <div slot="text">{$_("markAllRead")}</div>
-                    </MenuItem>
+
+                    {#if $isProposalGroup}
```

Comment:
This should only be enabled for CHAT proposals for now

Reply:
Yes - was on my mind to fix